root.go

```diff
@@ -47,7 +47,7 @@ func rootCommand() *cobra.Command {
 
 	cmd.AddCommand(create)
 	cmd.AddCommand(command.BuildCmd())
-	cmd.AddCommand(command.DevCmd())
+	// cmd.AddCommand(command.DevCmd())
```

Comment:
Can you add a TODO here?
```suggestion
        // TODO: Re-enable when dev os updated to work with e2core
	// cmd.AddCommand(command.DevCmd())
```